### ISSUE TITLE
About the training loss function

### ISSUE THREAD

**Issue by Xiao-congxi** [google-research/timesfm#188]
During the pre-training, is the used loss function a combination of the MSE loss calculated on the first [mean] output head and the quantile loss calculated on other 9 [quantile] output heads?

**Comment by rajatsen91** [comment]
Yes that is what we have been using.

**Comment by Xiao-congxi** [comment]
Thank you very much for your immediate reply. I have another question about the experiment results in the paper. Why are the MAE of the fine-tuned TimesFM (Table 2) is higher than the zero-shot results (Table 5) ?

**Comment by ani0135** [comment]
@Xiao-congxi Can you please provide the loss function used here which is the combination of both heads?
